pkgs/by-name/go/goframe-cli/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

I believe the `buildGoModule` builder now supports the `finalAttrs` pattern.